MERLin/MERLin/Deeplinking/Deeplink.swift

```diff
@@ -89,6 +89,11 @@ public extension Deeplinkable {
             let range = Range(match.range(at: 0), in: deeplink) else { return nil }
         
         let remainder = String(deeplink.suffix(from: range.upperBound))
+            .split(separator: "/")
+            .filter { !$0.isEmpty }
+            .map(String.init)
+            .joined(separator: "/")
+        
         guard remainder.count > 0 && remainder != "/" else { return nil }
         
         let resultingDeeplink = "\(schema)://\(remainder)".replacingOccurrences(of: "///", with: "//")
```

Comment:
`replacingOccurrences(of: "///", with: "//")` looks redundant after above
And do we have unit test for this?

Reply:
We have DeeplinkableTests which covers getting remainderDeeplinks.

I believe you are correct and that the `replacingOccurrences` can be removed now.